### PR TITLE
Latest support tickets

### LINKED ISSUE #27: Make Latest Orders component work
[Details](https://trello.com/c/2zXq27gE/11-make-latest-orders-component-work)

### PR DESCRIPTION
**Pull request by mmirek98**
Closes #27

### CODE REVIEW

File: src/views/Tickets/components/TicketsCard/TicketsCard.js [hunk 1]
```diff
@@ -68,73 +68,77 @@ const TicketsCard = ({ data, isTrip }) => {
...
-          <Avatar aria-label="recipe" className={classes.avatar}>
-            {`${content.firstName[0]}${content.lastName[0]}`}
-          </Avatar>
-        }
-        action={<CloseEditButton id={id} />}
-        title={`${content.firstName} ${content.lastName}`}
-        subheader={new Date().toLocaleString()}
-      />
+    <>
+    { !ticket.closed &&
```
Comment (mmirek98) [inline src/views/Tickets/components/TicketsCard/TicketsCard.js:72]: Before adding this line, there was error while navigating to `/support` after fetching tickets in `LatestSupportTickets` component (`dashboard`): 
image

Error was thrown becouse tickets with closed status have `content` property equals to: `tickets.content === 'Close ticket'`

File: src/views/Dashboard/components/LatestSupportTickets/LatestSupportTickets.js [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+                      <Tooltip enterDelay={300} title="Sort">
+                        <TableSortLabel
+                          active
+                          direction={sortByDateDesc ? 'desc' : 'asc'}
+                          onClick={handleSortTicketsByDate}>
+                            Date
+                        </TableSortLabel>
+                      </Tooltip>
+                    </TableCell>
+                    <TableCell sortDirection="desc">
```
Comment (sebapaszynski) [inline src/views/Dashboard/components/LatestSupportTickets/LatestSupportTickets.js]: If you want, you can add the padding = {'checkbox'} parameter here, but if you like the current look more, ignore this comment 😁

**Review by sebapaszynski — APPROVED** [review]
Looks very good 😎

File: src/views/Dashboard/components/LatestSupportTickets/LatestSupportTickets.js (hunk 2 above)
Comment (mmirek98) [reply src/views/Dashboard/components/LatestSupportTickets/LatestSupportTickets.js]: Ok, looks better, added :+1:

**Pull request merged**